docs/UserGuide.md

```diff
@@ -177,9 +193,13 @@ t/friends tg/@yuyubern pf/Graphic Designer i/60000
 
 You should directly see the client profile added to the list of clients in the application window. After adding a client profile, you can now perform various operations on the client profile as specified in the next few sections!
 
+[↑ Back to Table of Contents](#table-of-contents)
+
 ### Adding a client interaction: ***interaction***
 
-After adding a client profile, you can now log your interactions with the client.  Use the ***interaction*** command to add an interaction to a client profile.
+After adding a client profile, their information is now recorded and easily accessible!
+But you may also want a way to log your interactions with the client. 
```

Comment:
"But" shouldn't be at the start of a sentence and it's not really negating the statement above it no? Maybe can rephrase this part

Reply:
fixed!